tests/virtualization/universal/guest_management.pm

```diff
@@ -19,6 +19,17 @@ use testapi;
 use utils;
 
 sub run {
+    ## Ensure every guest has <on_reboot>restart</on_reboot>
+    foreach my $guest (keys %virt_autotest::common::guests) {
+        my $filename = "$guest.xml";
+        assert_script_run("virsh dumpxml $guest > $filename");
+        if (script_run("! cat $filename | grep 'on_reboot' | grep -v 'restart'") != 0) {
+            record_info("$guest bsc#1153028", "Setting on_reboot=restart for $guest");
+            assert_script_run("sed -i '/\<on_reboot\>/c\  <on_reboot\>restart\</on_reboot\>/' $filename");
```

Comment:
@pdostal Those two lines have been forgotten and have been added now.

Reply:
👍